tests/AssetManagerTest/Controller/ConsoleControllerTest.php

```diff
@@ -55,6 +55,9 @@ public function setUp()
         $assetManager->setAssetCacheManager($assetCacheManager);
 
         $this->request = new ConsoleRequest();
+        if (!class_exists(RouteMatch::class)) {
```

Comment:
One thing you can do here is to create a helper method for creating a `RouteMatch` instance, allowing the ability to test on either v2 or v3:

```php
// at the top of the file:
use Zend\Mvc\Router\RouteMatch as V2RouteMatch;
use Zend\Router\RouteMatch;

// Within the class:
public function createRouteMatch(array $params = [])
{
    $class = class_exists(V2RouteMatch::class) ? V2RouteMatch::class : RouteMatch::class;
    return new $class($params);
}

// Within your setup:
$this->routeMatch = $this->createRouteMatch(['controller' => 'console']);
```